### DISCUSSION TITLE
Need help with local storage

### DISCUSSION THREAD

**Discussion by Marianna-Ikonomou** [neuefische/questions#554] [Web] [answered]
Hey, my problem with local storage is that I can see all the information after refreshing the page except the image. 
I appreciate any help. 
Here is my PR if you want to take a look:
https://github.com/Marianna-Ikonomou/Capstone---Find-my-Pet/pulls

**Comment by lenzloesche** [comment]
I think you have to convert the string of the blob url back to a blob url:
`const blobUrlString = localStorage.getItem('myBlobUrl');
const blobUrl = URL.createObjectURL(new Blob([blobUrlString]));`
I am not sure that this works. try it. :)

**Comment by mpagels** [comment, marked as answer]
Lenz is correct. You cannot store am image like a `.jpg` and/or `png` .etc in `localStorage`. Because you can only `strings` in `localStorage`

You should use a external service like [cloudinary](https://cloudinary.com/) where you can store the image and you will get an `url` as string and save that in `localStorage`

Take a look at this [awesome cloudinary next tutorial](https://cloudinary.com/guides/front-end-development/integrating-cloudinary-with-next-js)
